kasa/smart/modules/lighttransitionmodule.py

```diff
@@ -166,3 +170,15 @@ async def set_turn_off_transition(self, seconds: int):
             "set_on_off_gradually_info",
             {"off_state": {**self._turn_on, "duration": seconds}},
         )
+
+    async def _check_supported(self) -> bool:
+        """Additional check to see if the module is supported by the device."""
+        return True
```

Comment:
Should this have a real check in place?

Reply:
Removed this, must have left it in by accident.